### PR TITLE
docs: Add content pages [JOB-111256]

### PR DESCRIPTION
**Pull request by Aiden-Brine**
## Changes

Adds the content pages to the new docs site. Also modifies the routes for the design pages so that they can now be found at `/design/<name>` as opposed to `/content/design/<name>`.

## Testing

Verify the content landing page exists, that the various sub pages exist, and that content results show up in search. Also check that the routing still works on the various design pages.

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/site/src/layout/SearchBox.tsx [hunk 1]
```diff
@@ -74,34 +84,60 @@ export const SearchBox = ({
...
-                      key={key}
-                    />
-                  );
-                })}
-              </ContentCardWrapper>
-            </Content>
-          )}
-          {filteredDesignList.length > 0 && (
-            <Box padding={{ top: "largest" }}>
+          <Content spacing={"larger"}>
```
Comment (Aiden-Brine) [inline packages/site/src/layout/SearchBox.tsx:87]: I know Rebecca added the same change to her styles PR that isn't in yet. Will coordinate with her if end up merging first

File: packages/site/src/layout/SearchBox.tsx [hunk 2]
```diff
@@ -32,6 +32,16 @@ export const SearchBox = ({
   const ref = useRef<InputTextRef>(null);
   const [search, setSearch] = useState("");
 
+  const filteredContentList = useMemo(() => {
```
Comment (Aiden-Brine) [inline packages/site/src/layout/SearchBox.tsx:35]: We should come back and improve this so you don't have to copy and paste every time you want to add a new section to the search box

File: packages/site/src/layout/SearchBox.tsx (hunk 1 above)
Comment (taylorvnoj) [reply packages/site/src/layout/SearchBox.tsx]: Looks like it got dealt with in the merge conflict! 👍🏻

File: packages/site/src/layout/SearchBox.tsx (hunk 2 above)
Comment (taylorvnoj) [reply packages/site/src/layout/SearchBox.tsx:35]: Good catch - do you mind creating a ticket for that and we can return to this later on?

File: packages/site/src/components/ContentLoader.tsx [hunk 3]
```diff
@@ -8,7 +8,13 @@ import { contentMap } from "../maps";
...
  */
 export const ContentLoader = () => {
-  const { name, type } = useParams<{ name: string; type: string }>();
+  let type = "content";
+  const { name } = useParams<{ name: string }>();
+  const location = useLocation();
+
+  if (location.pathname.includes("/design")) {
+    type = "design";
+  }
```
[the remark this answers is not in the document]
Comment (taylorvnoj) [reply packages/site/src/components/ContentLoader.tsx:17]: Aiden and I chatted and I'm up to speed on what's going on here. Components is handled differently and all other collections will make their way into this ContentLoader as we add them. Thanks Aiden!

**Review by taylorvnoj — APPROVED** [review]
LGTM! Re: your concerns about routing - I totally agree there can be some cleanup later.

Do you mind writing a cleanup ticket for that logic? And also a cleanup ticket for the SearchBox logic?

Otherwise, really great work @Aiden-Brine 🚢

**Pull request merged**